docs/api/nitrogen.md

```
@@ -0,0 +1,184 @@
+    fn allocate_contiguous_frames(&self, count: usize) -> Result<u64, DriverContextError>;
+    fn map_mmio_region(&self, phys: usize, virt: usize, size: usize) -> Result<(), DriverContextError>;
+    fn map_page(&self, virt: usize, phys: usize, flags: PageFlags) -> Result<(), DriverContextError>;
+    fn free_frame(&self, phys: u64);
+    fn free_contiguous_frames(&self, phys: u64, count: usize);
+    fn dma_map(&self, device_id: u16, phys: u64, size: usize) -> Result<u64, DriverContextError>;
+    fn dma_unmap(&self, iova: u64, size: usize);
+}
+```
+
+**関連型**:
+| 型 | 役割 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)